bin/check_formatting.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:async';
+import 'dart:io';
+import 'package:io/io.dart';
+
+Future<Null> main() async {
```

Comment:
main should return void.

Reply:
I followed the first code example for the `ProcessManager` from here: https://pub.dartlang.org/packages/io. `SharedStdIn.terminate()` returns a `Future<Null>`.

Reply:
Ah okay, I didn't catch the use of async. I dunno why